### PR TITLE
fix: hide missing token warning when disabled

### PR DESCRIPTION
**Pull request by jycouet**
## Issue
When enabled is false, we get some logs like:
```
[hive][usage] token is missing
[hive][reporting] token is missing
```
I would expect that nothing happens if enabled is false.

## Tests
- [x] enabled false & debug true
  ✅ logging that: Hive is disabled
- [x] enabled false & debug false
  ✅ nothing shown

## Fix
Returning an empty obj from createHive if it's not enabled, like this, plugins will not do anything.

### CODE REVIEW

File: packages/libraries/client/tests/enabled.spec.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    info: jest.fn(),
+  };
+
+  const hive = createHive({
+    enabled: false,
+    debug: true,
+    agent: {
+      logger,
+    },
+    token: 'Token',
```
[the remark this answers is not in the document]
Comment (jycouet) [reply packages/libraries/client/tests/enabled.spec.ts:15]: It's to make ts happy that we put a value for testing.

It's also the case in actual tests: https://github.com/kamilkisiela/graphql-hive/blob/main/packages/libraries/client/tests/info.spec.ts#L15

How should we tackle this?

**Comment by kamilkisiela** [comment]
@jycouet 
I think we could reuse this:
https://github.com/kamilkisiela/graphql-hive/blob/main/packages/libraries/client/src/internal/operations-store.ts#L17-L28

https://github.com/kamilkisiela/graphql-hive/blob/main/packages/libraries/client/src/internal/reporting.ts#L14-L19

https://github.com/kamilkisiela/graphql-hive/blob/43f6817e1fd8a0ddf5e5680512474be77ddf2dbe/packages/libraries/client/src/internal/usage.ts#L39-L46

**Comment by jycouet** [comment]
@kamilkisiela I updated with your suggestion

**Comment by jycouet** [comment]
🤔 seems that there are a lot of breaks! 
Is it possible that it's a turbo cache issue?

I hate to say it but: _"it's working on my machine!"_

**Pull request merged**